### DISCUSSION TITLE
Chrome browser is not launching.

### DISCUSSION THREAD

**Discussion by sthallapelly** [anilabhadatta/educative.io_scraper#34] [Q&A] [answered]
Hi,
can you clarify step 5 under "To Run the project manually using git and python:" section? what is chrome-bin latest? you said download from latest releases, I'm new to this tech stack, no so sure from where I can download it. 
I followed the steps got below error:
<img width="679" alt="image" src="https://user-images.githubusercontent.com/18119504/222303572-78da692b-879b-468c-9be1-fbe7267bc286.png">

I tried to use my chrome.exe file copied from my program files but it doesn't work.
I see in your screenshot it chrome-bin-win.zip I'm not sure from where it need to download. tried internet without luck. Please help

**Comment by anilabhadatta** [comment, marked as answer]
@sthallapelly https://github.com/anilabhadatta/educative.io_scraper/releases/download/Educative-Scraper-v6.8/Chrome-bin-win.zip
Download and extract the folder, the path of chrome.exe should match with your path in screenshot .
Open two cmd, create virtual env and install required packages.
Make Sure you have python 3.9+.
run python chromedriver.py and python educative_scraper.py in those two terminals.

**Comment by anilabhadatta** [comment]
@sthallapelly any updates?

**Reply by sthallapelly** [reply]
Thanks it is workin

**Comment by anilabhadatta** [comment]
@sthallapelly please check latest repo, should fix all your problems.
